stan/math/prim/scal/prob/neg_binomial_lccdf.hpp

```diff
@@ -101,8 +101,8 @@ namespace stan {
         const T_partials_return alpha_dbl = value_of(alpha_vec[i]);
         const T_partials_return beta_dbl = value_of(beta_vec[i]);
         const T_partials_return p_dbl = beta_dbl / (1.0 + beta_dbl);
-        const T_partials_return d_dbl = 1.0 / ( (1.0 + beta_dbl)
-                                                * (1.0 + beta_dbl) );
+        const T_partials_return d_dbl = 1.0 / ((1.0 + beta_dbl)
```

Comment:
This isn't just form, but that `1 + beta_dbl` should be saved as a temp up top and reused:

```
const T_partials_return beta_dbl_plus_1 = beta_dbl + 1;
```

And we should be using `inv(x)` rather than `1 / x` where possible.